mypy.ini

```diff
@@ -1,4 +1,5 @@
 [mypy]
+implicit_optional = True
```

Comment:
Shouldn't it be  `no_implicit_optional = False` ? Also wouldn't it be better to refactor code?

Reply:
> Also wouldn't it be better to refactor code?

I could do that if you want, only reason why I didn't do it in the first place is that Optional is simply too long to be included in every parameter

Reply:
no, shorter is better =D just found only that one